frontend/src/App.css

```diff
@@ -60,6 +84,12 @@
 .ndl-data-grid-root .ndl-data-grid-navigation {
   padding-block: 5px !important;
 }
+.ndl-drawer .ndl-drawer-close-button {
+  background-color: rgb(245 246 246 / 1);
```

Comment:
You can't use static rgb here otherwise you are setting the same color for both light and dark theme 
Please use variables from the palette we provide (for example rgb(var(--theme-palette-neutral-bg-strong)); or another one that fit with the current design)

Reply:
Addressed the concerns